.github/workflows/system-tests.yml

```diff
@@ -144,6 +149,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: 'DataDog/system-tests'
+          ref: ${{ inputs.ref }}
```

Comment:
you'll need to populate this parameters to few more places : `parametric` (easy one the parameter already exists, you juts ned to pass it), `lib_injection`, `run_end_to_end` and `external_processing`